signac/contrib/project.py

```diff
@@ -1568,7 +1568,7 @@ def init_project(cls, name, root=None, workspace=None, make_dir=True):
 
     @classmethod
     def get_project(cls, root=None, search=True, **kwargs):
-        """Find a project configuration and return the associated project.
+        r"""Find a project configuration and return the associated project.
```

Comment:
is using raw strings everywhere a convention even when theyre not necessary?

Reply:
tl;dr: Using a raw string here is a non-obvious trick for properly documenting `**kwargs` with Sphinx.

This has `**kwargs` and the only way to make a valid docstring for Sphinx is to use `\*\*kwargs` below, and mark it as a raw string so that the backslashes aren't interpreted as escape characters. If you use `**kwargs` without the backslashes, Sphinx will interpret that as bold formatting. If you use `\*\*kwargs` without the raw string, then it will attempt to interpret those as escape sequences. If you use `\\*\\*kwargs` to escape the slashes without the raw string, I think something else breaks.